src/Date/JavaScriptFormatConverter.php

```diff
@@ -14,47 +14,30 @@
 namespace Sonata\Form\Date;
 
 /**
- * Handles Moment.js <-> PHP date format conversion.
- *
- * Inspired by https://github.com/fightbulc/moment.php/blob/master/src/Moment/CustomFormats/MomentJs.php
+ * Handles JavaScript <-> PHP date format conversion.
  *
  * @author Hugo Briand <briand@ekino.com>
  * @author Andrej Hudec <pulzarraider@gmail.com>
  */
-final class MomentFormatConverter
+class JavaScriptFormatConverter
```

Comment:
I have lost some stuff from the 2.x branch of this file. I should recover final here, and probably more stuff below.